packages/amplify-category-function/src/provider-utils/awscloudformation/service-walkthroughs/lambdaLayerWalkthrough.ts

```diff
@@ -51,31 +144,34 @@ function layerNameQuestion(context: any) {
   ];
 }
 
-function layerPermissionsQuestion() {
+function layerPermissionsQuestion(params: Partial<LayerParameters>) {
   return [
     {
-      type: 'list',
+      type: 'checkbox',
       name: 'layerPermissions',
       message: 'Who should have permission to use this layer?',
       choices: [
         {
           name: 'Only the current AWS account',
           value: Permissions.private,
+          checked: _.includes(params.layerPermissions, Permissions.private) ? true : false,
```

Comment:
ternary is redundant. `_.includes()` returns true / false

Reply:
fixed